src/components/DeviceSelector.tsx

```diff
@@ -56,11 +54,10 @@ export default () => {
         <DeviceSelector
             deviceSetupConfig={deviceSetupConfig}
             deviceListing={deviceListing}
-            onDeviceSelected={device => {
-                if (device.traits.jlink) {
-                    dispatch(setShowPPK1Dialog(true));
-                }
-            }}
+            deviceFilter={device =>
+                isDeviceInDFUBootloader(device) ||
+                device.usb?.device.descriptor.idProduct === 0xc00a
+            }
```

Comment:
not sure if we should change behavior here. kinda breaking change

Reply:
This make sure we only see PPK kits .... not sure what is breaking here

Reply:
We had users selecting the DK and not the PPK kit... this for me fixes that issue ....